scripts/insert_into_appointments.sql

```diff
@@ -1 +1,34 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(date, time, comment, doctor_id, patient_id)
+  SELECT '2016-08-12', '10:00', 'Internal Medicine appt', d.id, p.id
+  FROM doctors d, patients p
+    WHERE d.specialty = 'Internal Medicine'
+    AND d.given_name = 'Vicky'
+    AND d.surname = 'Hilton'
+    AND p.given_name = 'Marsha'
+    AND p.surname = 'Wilcox'
+  UNION
+  SELECT '2016-08-17', '13:00', 'Dermatology appt', d.id, p.id
+  FROM doctors d, patients p
+    WHERE d.specialty = 'Dermatology'
+    AND d.given_name = 'Spencer'
+    AND d.surname = 'Hubbard'
+    AND p.given_name = 'Ivory'
+    AND p.surname = 'Talley'
+  UNION
+  SELECT '2016-08-12', '10:00', 'Dermatology appt', d.id, p.id
+  FROM doctors d, patients p
+    WHERE d.specialty = 'Dermatology'
+    AND d.given_name = 'Spencer'
+    AND d.surname = 'Hubbard'
+    AND p.given_name = 'Marsha'
+    AND p.surname = 'Wilcox'
+  UNION
+  SELECT '2016-08-17', '13:00', 'Internal Medicine appt', d.id, p.id
+  FROM doctors d, patients p
+    WHERE d.specialty = 'Internal Medicine'
+    AND d.given_name = 'Vicky'
+    AND d.surname = 'Hilton'
+    AND p.given_name = 'Blake'
+    AND p.surname = 'Hodge'
+    ;
```

Comment:
This works fine, but you're doing too much work! The point of `WHERE` parameters is for you to narrow down the specific doctor. Yes, it is possible to have multiple Dr. Hiltons, who are all in Internal Medicine. But in the case, you could have just use the `surname` parameter, like this:

```sql
INSERT INTO appointments(date, time, doctor_id, patient_id)
  SELECT '20150812', '1000',
    (SELECT id FROM doctors WHERE surname = 'Hilton'),
    (SELECT id FROM patients WHERE surname = 'Wilcox')
  UNION
  ;
```